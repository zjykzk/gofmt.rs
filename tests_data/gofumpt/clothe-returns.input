package p

func foo() (err error) {
	if true {
		return
	}
	if false {
		return func() (err2 error) {
			return
		}
	}
	return
}

func bar() (_ int, err error) {
	return
}

func baz() (a, b, c int) {
	return
}

func qux() (file string, b int, err error) {
	if err == nil {
		return
	}

	// A comment
	return
}

// quux does quuxy things
func quux() {}
