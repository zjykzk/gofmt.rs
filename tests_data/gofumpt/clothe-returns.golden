package p

func foo() (err error) {
	if true {
		return err
	}
	if false {
		return func() (err2 error) {
			return err2
		}
	}
	return err
}

func bar() (_ int, err error) {
	return
}

func baz() (a, b, c int) {
	return a, b, c
}

func qux() (file string, b int, err error) {
	if err == nil {
		return file, b, err
	}

	// A comment
	return file, b, err
}

// quux does quuxy things
func quux() {}
